### ISSUE TITLE
Installing on Windows: can't perform  <gn args out\thorium>

### ISSUE THREAD

**Issue by Alex-Alex-A** [Alex313031/thorium#834]
Have an error , launching gn args out\thorium:

<The expected Rust version is 3cf924b934322fd7b514600a7dc84fc517515346-3-llvmorg-19-init-14561-gecea8371 but the actual version is None
Did you run "gclient sync"?>

Do you have an idea what is it?
![thorium-error](https://github.com/user-attachments/assets/539c44d7-79a9-4f6e-b660-5344121bdfb4)

**Comment by gz83** [comment]
https://github.com/Alex313031/Thorium-Win/releases

Please download the Windows version here without compiling

**Comment by Alex-Alex-A** [comment]
Thank you, may be I'll do. But I'd prefer to compile  it myself )

**Comment by gz83** [comment]
If you decide to compile it yourself, you may end up spending a long time doing it and it will take up a lot of your device's resources.

In addition, for the current Windows version you may also need a file that is currently only generated normally on Linux to fix errors encountered during the compilation process.

At the same time, you need to strictly refer to the instructions in the //docs directory to compile, and you will also face other problems, such as the instructions for the windows platform in the //docs directory need to be updated, and the sh file used by Thorium to process the chromium source code does not run well on windows, etc.

To sum up, please consider your actions carefully, but I will not stop you from compiling in the end, because it is your freedom.
